de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/versioning/JGitService.java

```diff
@@ -0,0 +1,97 @@
+package de.fu_berlin.inf.dpp.versioning;
```

Comment:
The name "versioning" is a bit confusing. Something like "vcs" (version control system) would be more clear.